File: CS/PS/복습/progr_lv1/두개뽑아서더하기.py
import itertools
def solution(numbers):
    answer = []
    combis = list(itertools.combinations(numbers, 2))
    s = set([])
    for i in combis:
        s.add(i[0] + i[1])
    lst_s = list(s)
    lst_s.sort()
    return list(lst_s)

File: CS/PS/Site/programmers/lv1/두개뽑아서더하기.js
function solution(numbers) {
  let numberSet = new Set();
  for (let i = 0; i < numbers.length; i++){
    for(let j = i + 1; j < numbers.length; j++){
      numberSet.add(numbers[i] + numbers[j]);
    }
  }
  return Array.from(numberSet).sort((a,b) => a-b);
}

console.log(solution([2,1,3,4,1]));
